lib/content-types/load.js

```diff
@@ -34,20 +34,23 @@ const load = () => {
       }
 
       const config = yaml.safeLoad(content);
-      const name = config.name;
-      const machine = kebab(config.name);
 
       if (!config.hasOwnProperty('name')) {
         reject(new Error('Content types require a name'));
       }
 
-      if (machineNames.indexOf(machine) === -1) {
-        machineNames.push(name);
-        config.machine = machine;
+      if (!config.hasOwnProperty('id')) {
+        reject(new Error('Content types require an id'));
+      }
+      const id = kebab(config.id);
```

Comment:
You're checking to see if the ID exists, but not checking to see that the ID the user has included equals the kebab'd version of the ID. You're just overriding that. We don't want to override it, we want to reject if it's not ===

```javascript
if (config.id !== kebab(config.id) {
  reject new Error(`${config.id} needs to be written in kebab case (e.g. ${kebab(config.id)}`);
}
````